gpflow/likelihoods/likelihoods.py

```diff
@@ -69,92 +69,296 @@ def inv_probit(x):
 
 
 class Likelihood(Module):
-    def __init__(self):
+    def __init__(self, num_latent_functions: int, num_data_dims: int):
```

Comment:
Can you clarify what num_data_dims specifies? I'm not sure it's the best name ...

Reply:
`num_latent_functions` and `num_data_dims` refer to the left most shape of F and Y respectively. And this for a give data point

I suggest to call them : `latent_dim` and `observation_dim` (no plural, it is a dimension)
 
@jameshensman @st-- ?

Reply:
*rightmost ;)

I don't mind what we call it. If we're going to be compatible with tfp we should prbably call it `event_dim` or `event_size`

Reply:
latent_dim & observation_dim sounds good

Reply:
(we have to keep in mind num_latent_gps != latent_dim ... it's complicated ...)